spatula/base.py

```diff
@@ -96,3 +96,43 @@ def convert_pdf(self, filename, type):
         data = pipe.read()
         pipe.close()
         return data
+
+
+class CSV(AbstractPage):
+    def __init__(self, scraper, url=None, *, obj=None, **kwargs):
+        super().__init__(scraper, url=url, obj=obj, **kwargs)
+        resp = self.do_request()
+        self.reader = csv.DictReader(io.StringIO(resp.text))
+
+    def do_request(self):
+        return self.scraper.get(self.url)
+
+    def scrape_page_items(self, page_type, url=None, **kwargs):
```

Comment:
it's late so maybe I'm missing something, but aren't these two functions inherited from AbstractPage?